kafka-ui-react-app/package.json

```diff
@@ -42,7 +46,8 @@
     "lint:fix": "eslint --ext .tsx,.ts src/ --fix",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "test:coverage": "jest --coverage"
```

Comment:
I think we can remove it because `react-scripts` provides `npm test -- --coverage`